getting-started-2.0.md

```diff
@@ -0,0 +1,155 @@
+# Getting Started with Helix
+
+This guide helps you as a web developer get started with Helix with very little effort, then progressively discover more features as you need them.
+
+## Step 1: Publish your content (look ma, no hands!)
+
+> You'll need:
+> - a [GitHub](https://github.com/join) account
+> - basic [Markdown](https://guides.github.com/features/mastering-markdown/) knowledge
+
+The easiest way to publish content is to have [_Helix Pages_](https://www.project-helix.page/) render and deliver a Markdown file in your GitHub repository:
+
+1. Click the _Create new repository_ button to set up a new public [GitHub repository](https://github.com/new/).
+   <br>![](./dummy.png "TODO: Create a repository")
+1. Click the _Create new file_ button in your GitHub repository to add a new file called `index.md`. This will be your homepage.
+   <br>![](./dummy.png "TODO: Create an index.md")
+1. Add some great Markdown content. Feel free to use [this example](https://raw.githubusercontent.com/adobe/helix-example/master/index.md "example index.md") for starters.
+1. Click the _Commit changes_ button.
+1. Go to `https://<my-repo>-<my_org>.project-helix.page/` to admire your work. Replace `<my-repo>` and `<my-org>` with your actual repository and organization names:
+   <br>![](./dummy.png "TODO: Helix Pages rendering your content")
+1. **Congratulations!** 🎉 You have just published your first page using _Helix Pages_!
+
+> **Hint:** You can add as many Markdown files as you like, and link them in Markdown.
+
+> **Note:** If your GitHub organization contains one or more dashes, add an extra dash between the repository and organization name in the _Helix Pages_ URL. For example, if your organization is called `my-org`, and your repository is `test`, the corresponding _Helix Pages_ URL would be `https://test--my-org.project-helix.page/`.
```

Comment:
I would clarify that _github organization_ and `gihub username` means the same here. over even better, don't use _organization_ as we target rather individuals as orgs, I guess.